app/views/admin/infobase/_order_form.html.haml

```diff
@@ -0,0 +1,23 @@
+%table.ui.very.basic.table
+  %thead
+    %tr
+      %th User Name
+      %th User Email
+      %th Delivery Address
+      %th Order Items
+      %th Date of Order
+      %th Price of Order
+  %tbody
+  - orders.includes(:user).each do |order|
+    %tr
+      %td #{order.user.name}
+      %td #{order.user.email}
+      %td #{order.address}
+      %td
+        - order.cart_items.includes(:menu).pluck(:title).each do |item|
```

Comment:
way too much logic on the view
`order.cart_items.includes(:menu).pluck(:title)`
forget about it.
View should be dumb and simply show variables that you give it.
So, this calculation go to controller, but as we talked action is also bad for this, so it goes straight to the facade. Easy as that :)